lib/modules/manager/conan/common.ts

```diff
@@ -0,0 +1,3 @@
+export function isComment(line: string): boolean {
+  return line.trim().startsWith('#');
```

Comment:
Can also start with `//` as comment, so handle that case

Reply:
No it cannot. Go see https://docs.conan.io/en/latest/reference/conanfile_txt.html#comments or the python doc